sync/downstream.py

```diff
@@ -1069,3 +1075,32 @@ def update_pr(git_gecko, git_wpt, sync, action, merge_sha, base_sha, merged_by=N
     except Exception as e:
         sync.error = e
         raise
+
+
+def get_temporary_metadata(sync):
```

Comment:
`temporary` seems like a bad name here; the point is that it's from GitHub.

I also wonder if it would be more like the other code to make this a method on the sync.